app/models/abstract-validation-model.js

```diff
@@ -25,9 +25,12 @@ export default class AbstractValidationModel extends Model {
 
   /**
    * Validate the model using the validation schema.
+   * @param {boolean} relaxMandatoryFoundingOrganization - if true, the
+   * requirement to specify a founding organisation is not enforced for OCMW
+   * associations and PEVAs
    * @returns {Promise<boolean>} - Whether the model is valid.
    */
-  async validate() {
+  async validate(relaxMandatoryFoundingOrganization) {
```

Comment:
**Suggestion (blocker):** `relaxMandatoryFoundingOrganization` is really specific. I'm pretty certain that we will need in the future to add more logic like that. What about passing an object instead of a boolean? It will increase the flexibility, and avoid a long list of arguments in the future :wink: 

**Proposal**
```javascript
  /**
   * Validate the model using the validation schema.
   * @param {object} [options] - The options for validation.
   * @returns {Promise<boolean>} - Whether the model is valid.
   */
  async validate( options = {} ) {
    this._validationError = undefined;
    const { attributes, relationships } = this.#serializeAll();
    const value = { ...attributes, ...relationships };

    try {
      await this.validationSchema.validateAsync(value, {
        abortEarly: false,
        context: {
          changedAttributes: this.changedAttributes(),
          ...options,
        },
      });
```
The validate method should be called `administrativeUnit.validate({relaxMandatoryFoundingOrganization: true})`